src/components/Heroes/rankColumns.ts

```diff
@@ -54,11 +54,29 @@ type Row = {
 type Column = ProColumn | PublicColumn | HeroColumn
 
 export const rankColumns = (props: Props) => {
-  return props.tabType === HeroesTab.PRO
-    ? generateProTabColumns(props.strings)
-    : generatePublicTabColumns(props.strings);
+  if (HeroesTab.PRO === props.tabType) return generateProTabColumns(props.strings);
```

Comment:
object syntax recommended: https://github.com/odota/web/pull/2702/files#diff-67254100eb551b3ec455e54f473fb9825b9d31a8d9c34cc332e109ce4f3b7ce3R58

Reply:
as in heroes.jsx: agreed, improves legibility.